app/javascript/test/ResultGrid.test.tsx

```diff
@@ -0,0 +1,57 @@
+import '@testing-library/jest-dom';
```

Comment:
Use below at top of this file to enable the mocking of fetch.
```
import { enableFetchMocks } from 'jest-fetch-mock';
enableFetchMocks();
```
You will need to install it by running `yarn add jest-fetch-mock --dev`